openedx.yaml

```diff
@@ -16,3 +16,4 @@ oeps:
     oep-18: True
     oep-30:
         applicable: False
+archived: True
```

Comment:
I'm guessing you did this one first and copied the change elsewhere?  Not thrilled with YAML being so arbitrary about capitalization and such...